test/runtime/chaos.go

```diff
@@ -71,7 +70,7 @@ var _ = Describe("RuntimeValidatedChaos", func() {
 		// We don't use -o jsonpath... here due to GH-2395.
 		//
 		// jq 'map(select(.status.identity.id != 4), del(.status.controllers, ..., (.status.identity.labels | sort)))'
-		filterHealthEP := fmt.Sprintf("select(.status.identity.id != %d)", identity.GetReservedID("health"))
+		filterHealthEP := "select(.status.identity.id != 4)" // ReservedIdentityHealth=4
```

Comment:
Can we declare a const, or use an existing variable instead of hard-coding 4

Reply:
I'm fine with a const as long as we don't use the one in the API definition. I'll add it.